specs/_features/maxeb_increase/capella.py

```diff
@@ -1146,10 +1156,14 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
     # Compute exit queue epoch
     exit_epochs = [v.exit_epoch for v in state.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
     exit_queue_epoch = max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])
-    exit_queue_churn = len([v for v in state.validators if v.exit_epoch == exit_queue_epoch])
-    if exit_queue_churn >= get_validator_churn_limit(state):
-        exit_queue_epoch += Epoch(1)
-
+    exit_balance_to_consume = validator.effective_balance
+    per_epoch_churn_limit = get_validator_churn_limit(state)
+    if state.exit_queue_churn + exit_balance_to_consume <= per_epoch_churn_limit:
+        state.exit_queue_churn += exit_balance_to_consume
+    else:  # Exit balance rolls over to subsequent epoch(s)
+        exit_balance_to_consume -= (per_epoch_churn_limit - state.exit_queue_churn)
+        additional_epochs, state.exit_queue_churn = divmod(exit_balance_to_consume - (per_epoch_churn_limit - state.exit_queue_churn), per_epoch_churn_limit)
```

Comment:
The exit churn leftover has already been subtracted from `exit_balance_to_consume` in the above line, shouldn't this computation be as follows?
```suggestion
        additional_epochs, state.exit_queue_churn = divmod(exit_balance_to_consume, per_epoch_churn_limit)
```

IMHO, fancy Python stuff hinders readability, maybe break it down into two separate computations?

Reply:
removed this in the latest version!